### DISCUSSION TITLE
Toggl Track Integration

### DISCUSSION THREAD

**Discussion by agluck91** [makeplane/plane#6191] [Ideas]
Right now in pro time tracking is done with manually logging work. All of my projects times get logged inside toggl. It would be awesome to have an integration that would allow me to some how link issues to toggl and get the time spent on each thing.

**Comment by agluck91** [comment]
If not a toggl integration something similar inside plane? I would be willing to get rid of a tool! HAHA 
Toggl does all the hard work for you though, and the integration APIs did not seem to crazy.

**Comment by danciaclara** [comment]
Thanks for sharing your workflow.

Plane has APIs and webhooks that you can use to create a custom integration with Toggl. You can check out our [API](https://developers.plane.so/api-reference/introduction) and [Webhooks](https://developers.plane.so/webhooks/intro-webhooks) documentation to get started.

Also, we’ve got time tracking enhancements on our radar, so stay tuned for updates. Could you also share what are the time tracking features in Toggl that you'd like to see in Plane? Thanks

**Reply by agluck91** [reply]
Oh nice. I did not know that documentation existed. I think I can does some triggers based on web hooks. Thank you!!

**Reply by agluck91** [reply]
I did not read all the words. I would love to see a start timer option. The least friction to start it the better. I love that toggl and othe time trackers have a cli, api, chrome extension to kick off a timer. I am not sure it fits in plane, but my timer apps will create an invoice for me so I can bill my clients.
